api/fittings/lisk_error_handler.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+var debug = require('debug')('swagger:lisk:error_handler');
+var util = require('util');
+
+module.exports = function create (fittingDef, bagpipes) {
+
+	debug('config: %j', fittingDef);
```

Comment:
debug how? TO log, or console?

Can we leave some comments about the % stuff

Reply:
The logic is happening by this line: 

`var debug = require('debug')('swagger:lisk:error_handler');`

So while running app we pass these flags to `DEBUG` variable it will output these logs. 

```
DEBUG=swagger:lisk:error_handler npm start
```

To see all swagger related debug information:

```
DEBUG=swagger* npm start
```

Reply:
ok great thank you

Reply:
can we get what you wrote above documented somewhere? In the readme or somewhere else, like docs.lisk.io?